biz.aQute.bndlib/src/aQute/bnd/osgi/resource/CapReq.java

```diff
@@ -20,26 +22,27 @@
 	private final Resource				resource;
 	private final Map<String,String>	directives;
 	private final Map<String,Object>	attributes;
+	private transient int				hashCode	= 0;
```

Comment:
I've never heard of `transient` being used that (a) way before. My assumption when looking at a field is that if it's not `final`, it can be reassigned (recomputed).